### PR TITLE
DBZ-735 Verify Avro serialization

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-735

### CODE REVIEW

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDecimalIT.java [hunk 1]
```diff
@@ -49,6 +49,7 @@ public void beforeEach() {
         DATABASE.createAndInitialize();
         initializeConnectorTestFramework();
         Testing.Files.delete(DB_HISTORY_PATH);
+        skipAvroValidation();
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDecimalIT.java:52]: Can you add a comment here about the reasoning? I see it's mentioned below, but I feel calling such method should justified always at the call site itself as there may be different reasons.

File: debezium-core/src/main/java/io/debezium/data/SchemaUtil.java [hunk 2]
```diff
@@ -119,6 +121,61 @@ public static String asDetailedString(SourceRecord record) {
...
     }
 
+    /**
+     * Sets the version of a passed schema to a new value.
+     *
+     * @param schema the schema to be updated
+     * @param version the target version value
+     * @return the new schema with the same structure but updated version
+     */
+    public static Schema setVersion(Schema schema, Integer version) {
```
Comment (gunnarmorling) [inline debezium-core/src/main/java/io/debezium/data/SchemaUtil.java:131]: Move to test code if only used there?

**Review by gunnarmorling — COMMENTED** [review]
Two comments inline.

**Comment by jpechane** [comment]
@gunnarmorling Done

**Pull request merged**